server/models/book.js

```diff
@@ -16,7 +16,12 @@ module.exports = (sequelize, DataTypes) => {
   Book.init({
     title: DataTypes.STRING,
     author: DataTypes.STRING,
-    rating: DataTypes.INTEGER
+    rating: DataTypes.INTEGER,
```

Comment:
`rating` was replaced by `averageRating` and `maturityRating` in the version of the model in the `master` branch.